control_panel_api/tests/permissions/test_s3bucket_permissions.py

```diff
@@ -28,18 +29,20 @@ def setUp(self):
             "control_panel_api.S3Bucket", name="test-bucket-1")
         self.s3bucket_2 = mommy.make(
             "control_panel_api.S3Bucket", name="test-bucket-2")
+        # Normal user has access to s3bucket_1
```

Comment:
Don't need comment.

Reply:
I still think that these comments can help understand the tests but I'll remove for the love of peace 👍